app/controllers/api/v2/validity_dates_controller.rb

```diff
@@ -0,0 +1,31 @@
+module Api
+  module V2
+    class ValidityDatesController < ApiController
```

Comment:
So this isn't really resourceful. How do the jsonapi docs say we should do this? Should we just surface the validity dates on the resources as they are currently?